### PR TITLE
DAOS-10258 dfs: Add debugging to posix_meta_entry test.

### PR DESCRIPTION
**Pull request by ashleypittman**
Test-tag: dm_posix_meta_entry_dcp

Required-githooks: true

Signed-off-by: Ashley Pittman <ashley.m.pittman@intel.com>

### CODE REVIEW

File: src/tests/ftest/datamover/posix_meta_entry.yaml [hunk 1]
```diff
@@ -21,6 +21,8 @@ container:
   type: POSIX
   control_method: daos
 dcp:
+  env_vars:
+    - D_LOG_MASK=INFO,DFS=DEBUG
```
Comment (daltonbohning) [inline src/tests/ftest/datamover/posix_meta_entry.yaml:25]: We want to keep this turned on?

File: src/tests/ftest/util/data_mover_test_base.py [hunk 2]
```diff
@@ -948,6 +948,11 @@ def run_datamover(self, test_desc=None,
         if test_desc is not None:
             self.log.info("Running %s: %s", self.tool, test_desc)
 
+        env = EnvironmentVariables.from_list(
+            self.params.get("env_vars", "/run/{}/*".format(self.tool.lower()), []))
+        if "D_LOG_FILE" not in env:
+            env["D_LOG_FILE"] = get_log_file("{}.log".format(self.tool.lower()))
```
Comment (daltonbohning) [inline src/tests/ftest/util/data_mover_test_base.py:954]: I don't expect this logic to fail, but I didn't verify on other tests. If we want to land with this logic, I think we should also run some sanity tests:
```
Test-tag: datamover,vm
```

File: src/tests/ftest/util/ior_test_base.py [hunk 3]
```diff
@@ -413,11 +413,8 @@ def _execute_command(self, command, fail_on_err=True, display_output=True, hosts
...
-            hosts = [str(
-                nodes) for code, nodes in list(
-                    result.items()) if code != 0]
-            raise CommandFailure(
-                "Error running '{}' on the following hosts: {}".format(
-                    command, NodeSet(",".join(hosts))))
+        if (0 not in result or len(result) > 1) and fail_on_err:
+            hosts = [str(nodes) for code, nodes in list(result.items()) if code != 0]
+            raise CommandFailure("Error running '{}' on the following hosts: {}".format(
+                command, NodeSet(",".join(hosts))))
```
Comment (daltonbohning) [inline src/tests/ftest/util/ior_test_base.py:419]: This shouldn't break anything, but as a sanity check:
`Test-tag: ior`

File: src/tests/ftest/datamover/posix_meta_entry.yaml (hunk 1 above)
Comment (ashleypittman) [reply src/tests/ftest/datamover/posix_meta_entry.yaml:25]: Yes, DFS is quiet at debug anyway so the logs produced by this test are appropriate.

File: src/tests/ftest/util/ior_test_base.py (hunk 3 above)
Comment (ashleypittman) [reply src/tests/ftest/util/ior_test_base.py:419]: In addition to the above I assume?

**Review by daltonbohning — APPROVED** [review]

**Review by mchaarawi — APPROVED** [review]

**Pull request merged**